README.rst

```diff
@@ -1,8 +1,5 @@
-.. raw:: html
-
-    <p align="center">
-        <img src="https://raw.githubusercontent.com/pycqa/bandit/main/logo/logotype-sm.png" alt="Bandit">
-    </p>
+.. image:: https://raw.githubusercontent.com/pycqa/bandit/main/logo/logotype-sm.png
+    :alt: Bandit
```

Comment:
https://docutils.sourceforge.io/docs/ref/rst/directives.html#image Allows for `:align:` e.g., 

```rst
.. image:: https://raw.githubusercontent.com/pycqa/bandit/main/logo/logotype-sm.png
    :alt: Bandit logo
    :align: center
```

Reply:
Yeah, I initially tried using align but it seems GitHub doesn't support that tag.

https://github.com/github/markup/issues/163